connect/api/src/main/java/org/apache/kafka/connect/data/ConnectSchema.java

```diff
@@ -207,9 +207,14 @@ public Schema valueSchema() {
      * @param value value to test
      */
     public static void validateValue(Schema schema, Object value) {
+        validateValue(null, schema, value);
+    }
+
+    public static void validateValue(String name, Schema schema, Object value) {
         if (value == null) {
             if (!schema.isOptional())
-                throw new DataException("Invalid value: null used for required field");
+                throw new DataException("Invalid value null used for required field: \"" + name
```

Comment:
nits: not sure if we should `:` after `Invalid value` in the error message. Otherwise LGTM.

Thanks for the update.